OrbitService/OrbitAsioServer.cpp

```diff
@@ -24,7 +25,17 @@ void OrbitAsioServer::Run(std::atomic<bool>* exit_requested) {
   std::thread process_list_thread{
       [this, exit_requested] { ProcessListThread(exit_requested); }};
 
+  // make stdin non blocking
+  fcntl(STDIN_FILENO, F_SETFL, O_NONBLOCK);
+  // buffer for reading from stdin.
+  char buffer[10] = "";
```

Comment:
use power of 2 by default - 16/32, drop initialization/replace it by {};

why have you decided against grpc call?

Reply:
(done) 
I only considered grpc as a fallback, because it seemed to be the case that the EOF that I am looking for here is not passed through by the ssh server running on the instance. But this turned out to be false and now with the shutdown using EOF via stdin it is a lot more elegant to start _and_ stop OrbitService via ssh. Everything ssh can be packed together and work independently without needing to involve gprc